### PR TITLE
Add client

### PR DESCRIPTION
**Pull request by anarellano**

### CODE REVIEW

File: client/pages/components/teamsDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    onClick(team);
+  };
+
+  return (
+    <div className="">
+      <Dropdown isOpen={dropdownOpen} toggle={toggle} direction="down">
+        <DropdownToggle caret>{selectedTeam}</DropdownToggle>
+        <DropdownMenu {...args}>
+          {nbaTeams.map((nbaTeam, i) => (
+            <DropdownItem key={i} onClick={() => handleClick(nbaTeam.acronym)}>
```
Comment (evanugarte) [inline client/pages/components/teamsDropdown.tsx]: its best to use something other than the array index as the key, we can use some other unique value like the acronym

```suggestion
          {nbaTeams.map((nbaTeam) => (
            <DropdownItem key={nbaTeam.acronym} onClick={() => handleClick(nbaTeam.acronym)}>
```
read more about it here
https://stackoverflow.com/a/59518082

File: client/pages/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./components/globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
```
Comment (evanugarte) [inline client/pages/layout.tsx]: do we even need this? idk what it does

File: client/pages/table.tsx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+  //
+  const fetchPlayerData = async (
+    page: number,
+    limit: number,
+    nbaPlayer: string,
+    team: string,
+    position: string
+  ) => {
+    try {
+      let url = `http://localhost:8000/getPlayers?page=${page}&limit=${limit}`;
```
Comment (evanugarte) [inline client/pages/table.tsx]: look up how we can use the `URL` class built into javascript instead, like
```js
const a = new URL("/getPlayers", "http://localhost:8000")

a.searchParams.append("page", 1)
a.searchParams.append("limit", 1)
a.searchParams.append("nbaPlayer", 1)

console.log(a.href)
```

File: client/pages/table.tsx [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+            <th>STL</th>
+            <th>BLK</th>
+            <th>TOV</th>
+            <th>PF</th>
+            <th>PTS</th>
+          </tr>
+        </thead>
+        <tbody>
+          {players.map((player, i) => (
+            <tr key={i}>
```
Comment (evanugarte) [inline client/pages/table.tsx]: again about using something other than the array index for a key, but we can pick a custom key after we address the other coments

File: client/pages/components/teamsDropdown.tsx (hunk 1 above)
Comment (anarellano) [reply client/pages/components/teamsDropdown.tsx]: Thank you for the knowledge, I have fixed the index issue

File: client/pages/layout.tsx (hunk 2 above)
Comment (anarellano) [reply client/pages/layout.tsx]: took it out, default on next

File: client/pages/table.tsx (hunk 4 above)
Comment (anarellano) [reply client/pages/table.tsx]: Change the key to be players name

File: client/pages/components/teamsDropdown.tsx (hunk 1 above)
Comment (evanugarte) [reply client/pages/components/teamsDropdown.tsx]: looks the same, did you push the latest commit?

**Review by evanugarte — APPROVED** [review]

**Pull request merged**